tree/ntuple/v7/src/RFieldMeta.cxx

```diff
@@ -149,24 +149,55 @@ void ROOT::Experimental::RClassField::Attach(std::unique_ptr<RFieldBase> child,
    RFieldBase::Attach(std::move(child));
 }
 
-void ROOT::Experimental::RClassField::AddReadCallbacksFromIORules(const std::span<const ROOT::TSchemaRule *> rules,
-                                                                  TClass *classp)
+std::vector<const ROOT::TSchemaRule *> ROOT::Experimental::RClassField::FindRules(const RFieldDescriptor *fieldDesc)
```

Comment:
```suggestion
std::vector<const ROOT::TSchemaRule *> ROOT::Experimental::RClassField::FindRules(const RFieldDescriptor *sourceFieldDesc)
```

Reply:
As far as I understand, the `fieldDesc` "correspond[s] to the current RClassField instance", so it's not the source of a specific rule